core/variant/variant_op.h

```diff
@@ -266,7 +272,7 @@ class OperatorEvaluatorModNZ {
 		*r_ret = a % b;
 		r_valid = true;
 	}
-	static inline void validated_evaluate(const Variant *left, const Variant *right, Variant *r_ret) {
+	static inline void validated_evaluate(const Variant *left, const Variant *right, Variant *r_ret, bool &r_valid) {
 		*VariantGetInternalPtr<R>::get_ptr(r_ret) = *VariantGetInternalPtr<A>::get_ptr(left) % *VariantGetInternalPtr<B>::get_ptr(right);
 	}
```

Comment:
ModNZ is probably also affected. Also there seems to be other template specializations for Vector3i etc.